nbproject/project.properties

```diff
@@ -0,0 +1,3 @@
+files.encoding=UTF-8
```

Comment:
This `nbproject` folder must not be added to iTowns.

Reply:
thanks for your advice